nominees/notesmaster.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "NotesMaster",
+  "aliases": [
+    "Notesmaster Elearning",
+    "NM"
```

Comment:
I don't think NM can be used as an alias for this platform - it's too generic.

Reply:
@conradsp Ok, that is fine, we accept that.

Reply:
@nathanbaleeta - can you update and remove NM as an alias?

Reply:
@nathanbaleeta we are fine for you to remove NM as an alias.